Path Planner Trajectories

avakimc opened the pull request:
- Trajectory for AutoChargeStationSequence is "EngageOnly.path"
- Trajectory for AutoScoreAndBalance is "ScoreAndBalance.path"
- Path names have been added to the relevant autonomous programs to replace PLACEHOLDER_STRING

avakimc committed:
Path Planner Trajectories

- Trajectory for AutoChargeStationSequence is "EngageOnly.path"
- Trajectory for AutoScoreAndBalance is "ScoreAndBalance.path"
- Path names have been added to the relevant autonomous programs to replace PLACEHOLDER_STRING

balghane commented on src/main/deploy/pathplanner/2.5R Inside Start.path:
```diff
@@ -2,13 +2,13 @@
   "waypoints": [
```
Can you undo the changes in this one? I think it was unintended

balghane commented on src/main/deploy/pathplanner/EngageOnlyTraj.path:
```diff
@@ -0,0 +1,49 @@
+    {
+      "anchorPoint": {
+        "x": 2.2052479075196247,
+        "y": 2.66455746081135
```
For both of these trajectories, the robot is going to try to move sideways a little bit on the way onto the charge station. You can click the position and make sure the y position is exactly the same for the two positions. 

Similar for heading, we should make the headings exactly 0 which you can do in the path planner application. They're pretty close to 0 but not quite.

avakimc committed:
Revisions to PathPlanner Trajectories for

- fixed 2.5R Inside Start
- changed heading to 0 for both paths
- made the paths straight lines by setting y-value to be the same

balghane commented on src/main/java/frc/robot/commands/AutoChargeStationSequence.java:
```diff
@@ -9,14 +9,13 @@
   private PathPlannerTrajectory traj =
       PathPlanner.loadPath(
-          Constants.PLACEHOLDER_STRING,
+          "EngageOnlyTraj.path",
```
I was looking at the examples and I think this doesn't need the file extension, see here: https://github.com/mjansen4857/pathplanner/wiki/PathPlannerLib:-Java-Usage

avakimc committed:
Second Revisions to PathPlanner Trajectories

- removed file extensions in path names

avakimc commented:
/format

github-actions[bot] committed:
Formatting fixes
